aten/src/ATen/native/CPUFallback.cpp

```diff
@@ -0,0 +1,141 @@
+#include <ATen/native/CPUFallback.h>
+
+#include <ATen/core/ivalue.h>
+#include <ATen/core/stack.h>
+#include <ATen/core/boxing/KernelFunction.h>
+#include <ATen/core/dispatch/Dispatcher.h>
+#include <torch/library.h>
+#include <ATen/Functions.h>
+#include <torch/library.h>
+
+namespace at { namespace native {
+
+// convenience helper for converting tensors to cpu
+
+std::vector<at::Tensor> to_cpu(const at::TensorList& tensors) {
```

Comment:
Perhaps the stock `at::to_cpu` should just support undefined? IDK...

Reply:
Agreed, I'll try adding that in. At some point I was debugging some failures related to undefined tensor and probably put the logic directly here out of frustration / to be extra safe 😛.